adoc/chapters/programming_interface.adoc

```diff
@@ -6475,10 +6487,12 @@ following values for the [code]#AccessMode# template parameter:
     | Access a buffer from a <<host-task>>.
 |====
 
-Programs which specify the access target as [code]#target::device# and then use
-the [code]#accessor# from a <<host-task>> are ill formed.  Likewise, programs
-which specify the access target as [code]#target::host_task# and then use the
-[code]#accessor# from a <<sycl-kernel-function>> are ill formed.
+Programs which specify the access target as [code]#target::device# and then
+capture the [code]#accessor# in a <<host-task>> can only use the accessor for
+interoperability through the [code]#interop_handle#.
+
+Programs which specify the access target as [code]#target::host_task# and then
+use the [code]#accessor# from a <<sycl-kernel-function>> are ill formed.
```

Comment:
```suggestion
use the [code]#accessor# from a <<sycl-kernel-function>> result in undefined behavior.
```

It's probably better to  be consistent with the previous paragraph and say this  is UB vs. ill formed.  Does the term "ill formed" mean that the compiler is required to issue a diagnostic?